chunktype.go

```diff
@@ -24,6 +24,7 @@ const (
 	CWR              chunkType = 13
 	SHUTDOWNCOMPLETE chunkType = 14
 	RECONFIG         chunkType = 130
+	FORWARDTSN       chunkType = 192
```

Comment:
Should we take a chance to rename these? All caps is both against _most_ of golangs style (besides posix things, etc.) and also give it unnecessary global scope

Reply:
Fixed at e928d.  I prefixed with 'ct' (chunk type).  E.g. ctForwardTSN